### DISCUSSION TITLE
”哔哩哔哩“被“哔哩”模仿“哔哩”被“哩哩”模仿！！！完美套娃

### DISCUSSION THREAD

**Discussion by Q-Tai-mu** [Richasy/Bili.Uwp#308] [常规]
![lili](https://user-images.githubusercontent.com/58099119/134774603-ba08a8b7-6117-442c-a98a-f48b7e185cc3.png)

**Comment by X0980** [comment]
这是？

**Reply by Q-Tai-mu** [reply]
PS图片
